ui.go

```diff
@@ -813,10 +816,23 @@ func (ui *ui) draw(nav *nav) {
 		}
 
 		ui.menuWin.printLine(ui.screen, 0, 0, st.Bold(true), lines[0])
+
 		for i, line := range lines[1:] {
 			ui.menuWin.printLine(ui.screen, 0, i+1, st, "")
 			ui.menuWin.print(ui.screen, 0, i+1, st, line)
 		}
+
+		// Append the current menu selection item to the actual left command value
+		cmdValue := string(ui.cmdAccLeft)
+		ui.msgWin.print(ui.screen, len(ui.cmdPrefix), 0, st, cmdValue)
+
+		leftLen := len(ui.cmdPrefix) + runeSliceWidth(ui.cmdAccLeft)
+		ui.msgWin.print(ui.screen, leftLen, 0, st, string(ui.cmdAccRight))
+
+		cursorX := ui.msgWin.x + len(ui.cmdPrefix) + runeSliceWidth(ui.cmdAccLeft)
+		cursorY := ui.msgWin.y
+
+		ui.screen.ShowCursor(cursorX, cursorY)
```

Comment:
@tarasyarema Above lines redundant?

Reply:
I have to look at it, does the workflow prints the command line row later on this execution?